package.json

```diff
@@ -7,11 +7,15 @@
   "main": "dist/server.js",
   "repository": "github:bluebrain/nexus-explorer",
   "url": "https://github.com/BlueBrain/nexus/issues/new?labels=fusion",
-  "type": "module",
   "scripts": {
-    "dev": "rm -rf ./dist && concurrently 'vite build -c ./vite.config.refresh.js' 'vite' 'node ./server'",
-    "build": "rm -rf ./dist && vite build -c ./vite.config.refresh.js && vite build",
-    "preview": "cross-env NODE_ENV=production node server",
+    "clean": "rm -rf ./dist",
+    "dev:client": "yarn clean && concurrently 'vite build -c ./vite.config.refresh.js' 'vite'",
+    "dev:server": "NODE_ENV=development ts-node-esm --project tsconfig.node.json -- ./server/index.ts",
+    "dev": "concurrently 'yarn dev:client' 'yarn dev:server'",
+    "build:server": "etsc --config=etsc.config.js",
+    "build:client": "vite build -c ./vite.config.refresh.js && vite build",
+    "build": "yarn clean && yarn build:client && yarn build:server",
+    "preview": "cross-env NODE_ENV=production node ./dist/server",
```

Comment:
Update dev and build scripts
in dev: using `ts-node` for server
in prod: using `esbuild` (estc) for server